examples/trigonometry.R

```diff
@@ -0,0 +1,71 @@
+# https://processing.org/tutorials/trig/
+
+px <- 0
+py <- 0
+px2 <- 0
+py2 <- 0
+angle <- 0
+angle2 <- 0
+x <- 0
+x2 <- 0
+radius <- 50
+frequency <- 2
+frequency2 <- 2
```

Comment:
I write the long code because I am not sure how to write them concisely. I will update them when we begin to adopt R style guide.